robottelo/test.py

```diff
@@ -389,13 +394,27 @@ def _get_vm_list(cls):
         cls.vm_list = vm_list_string.split(',')
 
     @classmethod
-    def _set_testcase_parameters(cls, savepoint_name,
-                                 raw_file_name, stat_file_name):
+    def _set_testcase_parameters(cls, savepoint_name, raw_file_name,
+                                 stat_file_name, **kwargs):
```

Comment:
Why use `**kwargs`? You don't need an arbitrary set of named arguments. You need a very specific set of optional arguments: `raw_reg` and `stat_reg`.